pkg/oc/admin/diagnostics/diagnostics/client/run_diagnostics_pod.go

```diff
@@ -43,6 +45,17 @@ func (d *DiagnosticPod) Description() string {
 	return "Create a pod to run diagnostics from the application standpoint"
 }
 
+func (d *DiagnosticPod) Requirements() (client bool, host bool) {
+	return true, false
+}
+
+func (d *DiagnosticPod) AvailableParameters() []types.Parameter {
+	return []types.Parameter{
+		{ImageTemplateParam, "Image template to use in creating a pod", &d.ImageTemplate.Format, variable.NewDefaultImageTemplate().Format},
+		{LatestImageParam, "If true, when expanding the image template, use latest version, not release version", &d.ImageTemplate.Latest, false},
+	}
+}
+
 // CanRun is part of the Diagnostic interface; it determines if the conditions are right to run this diagnostic.
 func (d *DiagnosticPod) CanRun() (bool, error) {
 	if d.PreventModification {
```

Comment:
minor change to error message below:
running the diagnostic pod is an API change, which is prevented `because the --prevent-modification flag was specified`

Reply:
I like it, thanks!